### ISSUE TITLE
Outdated Korean contents in the dev-1.23-ko.1

### ISSUE THREAD

**Issue by seokho-son** [kubernetes/website#30845]
# This is a Bug Report
## Problem
Outdated files in the upstream/dev-1.23-ko.1 branch.

Report generated by the following command.
`scripts/diff_l10n_branches.py ko upstream/dev-1.22-ko.3 upstream/dev-1.23-ko.1`

### 93 files to be modified 
1. [x] M1.  `content/en/_index.html`  | 2(+XS) 2(-)
1. [x] M2.  `content/en/community/_index.html`  | 1(+XS) 0(-)
1. [x] M3.  `content/en/docs/concepts/architecture/cloud-controller.md`  | 2(+XS) 2(-)
1. [x] **M4.**  `content/en/docs/concepts/architecture/nodes.md`  | **102(+L) 13(-)**
1. [x] M5.  `content/en/docs/concepts/cluster-administration/addons.md`  | 6(+XS) 1(-)
1. [x] M6.  `content/en/docs/concepts/cluster-administration/networking.md`  | 2(+XS) 55(-)
1. [x] M7.  `content/en/docs/concepts/cluster-administration/system-logs.md`  | 63(+M) 5(-)
1. [x] M8.  `content/en/docs/concepts/configuration/manage-resources-containers.md`  | 8(+XS) 7(-)
1. [x] M9.  `content/en/docs/concepts/configuration/overview.md`  | 1(+XS) 1(-)
1. [x] M10.  `content/en/docs/concepts/configuration/secret.md`  | 1(+XS) 1(-)
1. [x] M11.  `content/en/docs/concepts/containers/container-lifecycle-hooks.md`  | 1(+XS) 1(-)
1. [x] M12.  `content/en/docs/concepts/containers/images.md`  | 68(+M) 0(-)
1. [ ] M13.  `content/en/docs/concepts/extend-kubernetes/_index.md`  | 4(+XS) 2(-)
1. [ ] M14.  `content/en/docs/concepts/extend-kubernetes/api-extension/custom-resources.md`  | 2(+XS) 2(-)
1. [ ] M15.  `content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md`  | 34(+M) 0(-)
1. [ ] M16.  `content/en/docs/concepts/extend-kubernetes/operator.md`  | 1(+XS) 3(-)
1. [x] M17.  `content/en/docs/concepts/overview/kubernetes-api.md`  | 54(+M) 2(-)
1. [x] M18.  `content/en/docs/concepts/scheduling-eviction/kube-scheduler.md`  | 1(+XS) 1(-)
1. [x] M19.  `content/en/docs/concepts/scheduling-eviction/scheduler-perf-tuning.md`  | 2(+XS) 2(-)
1. [x] M20.  `content/en/docs/concepts/services-networking/connect-applications-service.md`  | 1(+XS) 1(-)
1. [x] M21.  `content/en/docs/concepts/services-networking/dns-pod-service.md`  | 1(+XS) 1(-)
1. [x] M22.  `content/en/docs/concepts/services-networking/dual-stack.md`  | 3(+XS) 8(-)
1. [x] M23.  `content/en/docs/concepts/services-networking/ingress-controllers.md`  | 5(+XS) 5(-)
1. [x] M24.  `content/en/docs/concepts/services-networking/ingress.md`  | 89(+M) 16(-)
1. [x] M25.  `content/en/docs/concepts/services-networking/service-traffic-policy.md`  | 1(+XS) 1(-)
1. [x] M26.  `content/en/docs/concepts/services-networking/service.md`  | 2(+XS) 2(-)
1. [x] M27.  `content/en/docs/concepts/storage/persistent-volumes.md`  | 42(+M) 13(-)
1. [x] M28.  `content/en/docs/concepts/storage/storage-capacity.md`  | 1(+XS) 2(-)
1. [x] M29.  `content/en/docs/concepts/storage/storage-classes.md`  | 2(+XS) 2(-)
1. [x] M30.  `content/en/docs/concepts/storage/volume-pvc-datasource.md`  | 1(+XS) 1(-)
1. [x] M31.  `content/en/docs/concepts/storage/volume-snapshot-classes.md`  | 1(+XS) 1(-)
1. [x] M32.  `content/en/docs/concepts/storage/volume-snapshots.md`  | 1(+XS) 1(-)
1. [x] M33.  `content/en/docs/concepts/storage/volumes.md`  | 82(+M) 159(-)
1. [x] M34.  `content/en/docs/concepts/workloads/controllers/cron-jobs.md`  | 23(+S) 14(-)
1. [x] M35.  `content/en/docs/concepts/workloads/controllers/deployment.md`  | 25(+S) 9(-)
1. [x] M36.  `content/en/docs/concepts/workloads/controllers/job.md`  | 55(+M) 6(-)
1. [x] **M37.**  `content/en/docs/concepts/workloads/controllers/statefulset.md`  | **109(+L) 9(-)**
1. [x] M38.  `content/en/docs/concepts/workloads/controllers/ttlafterfinished.md`  | 22(+S) 29(-)
1. [x] M39.  `content/en/docs/concepts/workloads/pods/disruptions.md`  | 2(+XS) 2(-)
1. [x] M40.  `content/en/docs/concepts/workloads/pods/ephemeral-containers.md`  | 1(+XS) 10(-)
1. [x] M41.  `content/en/docs/concepts/workloads/pods/pod-lifecycle.md`  | 72(+M) 42(-)
1. [x] M42.  `content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md`  | 9(+XS) 5(-)
1. [x] M43.  `content/en/docs/contribute/advanced.md`  | 5(+XS) 5(-)
1. [x] M44.  `content/en/docs/contribute/new-content/_index.md`  | 3(+XS) 1(-)
1. [x] M45.  `content/en/docs/contribute/participate/pr-wranglers.md`  | 8(+XS) 1(-)
1. [x] M46.  `content/en/docs/contribute/review/for-approvers.md`  | 1(+XS) 1(-)
1. [x] M47.  `content/en/docs/reference/_index.md`  | 2(+XS) 2(-)
1. [x] M48.  `content/en/docs/reference/access-authn-authz/authorization.md`  | 1(+XS) 1(-)
1. [x] M49.  `content/en/docs/reference/command-line-tools-reference/feature-gates.md`  | 73(+M) 23(-)
1. [x] M50.  `content/en/docs/reference/command-line-tools-reference/kube-proxy.md`  | 11(+S) 95(-)
1. [x] M51.  `content/en/docs/reference/glossary/pod-disruption.md`  | 7(+XS) 2(-)
1. [x] M52.  `content/en/docs/reference/glossary/quantity.md`  | 3(+XS) 2(-)
1. [x] M53.  `content/en/docs/reference/glossary/secret.md`  | 1(+XS) 1(-)
1. [x] M54.  `content/en/docs/reference/kubectl/cheatsheet.md`  | 25(+S) 7(-)
1. [x] M55.  `content/en/docs/reference/kubectl/overview.md`  | 1(+XS) 1(-)
1. [x] M56.  `content/en/docs/reference/labels-annotations-taints.md`  | 30(+M) 5(-)
1. [x] **M57.**  `content/en/docs/reference/scheduling/config.md`  | **191(+L) 27(-)**
1. [x] M58.  `content/en/docs/reference/scheduling/policies.md`  | 6(+XS) 95(-)
1. [x] M59.  `content/en/docs/reference/tools/_index.md`  | 16(+S) 0(-)
1. [x] M60.  `content/en/docs/reference/using-api/client-libraries.md`  | 1(+XS) 0(-)
1. [x] M61.  `content/en/docs/reference/using-api/health-checks.md`  | 4(+XS) 4(-)
1. [x] M62.  `content/en/docs/setup/production-environment/container-runtimes.md`  | 1(+XS) 1(-)
1. [x] M63.  `content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md`  | 1(+XS) 1(-)
1. [x] M64.  `content/en/docs/setup/production-environment/tools/kubespray.md`  | 5(+XS) 5(-)
1. [x] M65.  `content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md`  | 33(+M) 4(-)
1. [x] M66.  `content/en/docs/setup/production-environment/windows/user-guide-windows-containers.md`  | 15(+S) 1(-)
1. [x] M67.  `content/en/docs/tasks/access-application-cluster/access-cluster.md`  | 1(+XS) 109(-)
1. [x] M68.  `content/en/docs/tasks/access-application-cluster/connecting-frontend-backend.md`  | 1(+XS) 1(-)
1. [x] M69.  `content/en/docs/tasks/access-application-cluster/list-all-running-container-images.md`  | 2(+XS) 2(-)
1. [x] M70.  `content/en/docs/tasks/administer-cluster/access-cluster-services.md`  | 12(+S) 1(-)
1. [x] M71.  `content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md`  | 1(+XS) 1(-)
1. [x] M72.  `content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md`  | 1(+XS) 1(-)
1. [x] M73.  `content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md`  | 16(+S) 0(-)
1. [x] M74.  `content/en/docs/tasks/administer-cluster/sysctl-cluster.md`  | 12(+S) 0(-)
1. [x] M75.  `content/en/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md`  | 6(+XS) 0(-)
1. [x] M76.  `content/en/docs/tasks/configure-pod-container/pull-image-private-registry.md`  | 22(+S) 15(-)
1. [x] M77.  `content/en/docs/tasks/debug-application-cluster/debug-running-pod.md`  | 1(+XS) 7(-)
1. [x] M78.  `content/en/docs/tasks/debug-application-cluster/resource-metrics-pipeline.md`  | 9(+XS) 0(-)
1. [x] M79.  `content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md`  | 7(+XS) 10(-)
1. [x] M80.  `content/en/docs/tasks/job/parallel-processing-expansion.md`  | 5(+XS) 5(-)
1. [x] M81.  `content/en/docs/tasks/network/validate-dual-stack.md`  | 4(+XS) 1(-)
1. [x] **M82.**  `content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md`  | **110(+L) 60(-)**
1. [x] **M83.**  `content/en/docs/tasks/run-application/horizontal-pod-autoscale.md`  | **203(+L) 137(-)**
1. [x] M84.  `content/en/docs/tasks/tls/managing-tls-in-a-cluster.md`  | 13(+S) 3(-)
1. [x] M85.  `content/en/docs/tasks/tools/included/optional-kubectl-configs-zsh.md`  | 1(+XS) 1(-)
1. [x] M86.  `content/en/docs/tasks/tools/install-kubectl-linux.md`  | 1(+XS) 1(-)
1. [x] M87.  `content/en/docs/tasks/tools/install-kubectl-macos.md`  | 1(+XS) 1(-)
1. [x] M88.  `content/en/docs/tasks/tools/install-kubectl-windows.md`  | 3(+XS) 4(-)
1. [x] M89.  `content/en/docs/tutorials/_index.md`  | 5(+XS) 0(-)
1. [x] M90.  `content/en/docs/tutorials/clusters/apparmor.md`  | 2(+XS) 2(-)
1. [x] M91.  `content/en/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html`  | 1(+XS) 1(-)
1. [x] M92.  `content/en/docs/tutorials/services/source-ip.md`  | 1(+XS) 1(-)
1. [x] M93.  `content/en/docs/tutorials/stateful-application/zookeeper.md`  | 1(+XS) 2(-)

### 2 files to be deleted 
1. [x] D1.  `content/en/docs/tasks/administer-cluster/enabling-topology-aware-hints.md`
1. [x] D2.  `content/en/docs/tasks/administer-cluster/highly-available-control-plane.md`

## Proposed Solution

Use `git diff` to check what is changed in the upstream. And apply the upstream changes manually 
to the `content/ko` of `upstream/dev-1.23-ko.1` branch.

For example:
```
# checkout `upstream/dev-1.23-ko.1`
...
# check what is updated in the upstream 
git diff upstream/dev-1.22-ko.3 upstream/dev-1.23-ko.1 -- content/en/_index.html
# apply changes to content/ko
vi content/ko/_index.html
...
# commit and push
...
# make PR to `upstream/dev-1.23-ko.1`
```

## Pages to Update

Pages in content/ko

**Comment by jihoon-seo** [comment]
[Preview]
> [ ] **M4.** `content/en/docs/concepts/architecture/nodes.md` | **102(+L) 13(-)**

<details>
  <summary>Captured image</summary>

![image](https://user-images.githubusercontent.com/46767780/145507470-7f64a767-c12c-47b2-a6db-f5da81f3173c.png)
</details>

---

> [ ] **M37.** `content/en/docs/concepts/workloads/controllers/statefulset.md` | **109(+L) 9(-)**

<details>
  <summary>Captured image</summary>

![image](https://user-images.githubusercontent.com/46767780/145507635-f64a5cd9-f8fc-49d8-9e48-87ba38f1d8ce.png)
</details>

---

> [ ] **M57.** `content/en/docs/reference/scheduling/config.md` | **191(+L) 27(-)**

<details>
  <summary>Captured image</summary>

![image](https://user-images.githubusercontent.com/46767780/145507872-e0b1d2fd-7f08-4343-8f77-3ab22b0adb24.png)
</details>

---

> * [ ]  **M82.**  `content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md`  | **110(+L) 60(-)**
> * [ ]  **M83.**  `content/en/docs/tasks/run-application/horizontal-pod-autoscale.md`  | **203(+L) 137(-)**

<details>
  <summary>Captured image</summary>

![image](https://user-images.githubusercontent.com/46767780/145508152-dcf53b72-ff82-4bb6-a44f-8fef025c3ed7.png)

![image](https://user-images.githubusercontent.com/46767780/145508205-88d88b0a-8816-4898-b0d5-9b50d7977d2d.png)
</details>

**Comment by jihoon-seo** [comment]
# 작업 진행 전 참고사항

- #30846
- #30845

두 이슈 모두에 포함되어 있는 파일도 있으니, 참고하시기 바랍니다.

[[비교 링크](https://github.com/jihoon-seo/k8s-website/commit/9eb402d5423b5565bd802e44b520080dbf9e7eda)]

**Comment by jihoon-seo** [comment]
M1-M12 제가 하겠습니다~
- M1-M12 : @jihoon-seo 
- ...

**Comment by gochist** [comment]
M13-M16 제가 해볼게요.

**Comment by seokho-son** [comment]
@gochist 님~~ M13-M16 만 처리하면 branch switching에 들어갈 수 있을 것 같습니다.
혹시 진행 중이신가요? 아니라면, 제가 맡아서 빠르게 처리할까 합니다. 그럼 의견 주세요. ^^

**Comment by gochist** [comment]
> @gochist 님~~ M13-M16 만 처리하면 branch switching에 들어갈 수 있을 것 같습니다.
> 혹시 진행 중이신가요? 아니라면, 제가 맡아서 빠르게 처리할까 합니다. 그럼 의견 주세요. ^^

제가 잊고서 진행을 막고 있었군요. 매우 죄송합니다. @seokho-son 님께서 진행 가능하실까요.

**Comment by seokho-son** [comment]
@gochist 아닙니다~ :) 제가 잘 못 챙기고 있었어요 ㅎㅎ

일단 제가 빠르게 처리해서 PR 올려보도록 하겠습니다.
이따가 리뷰 및 승인을 부탁 드릴게요! :)

**Comment by seokho-son** [comment]
@gochist #31617 올렸습니닷 ! :)

**Comment by jihoon-seo** [comment]
PR #31637 : merged 36 commits into `main` from `dev-1.23-ko.1`
/close
